ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/collector/BasicCollector.java

```diff
@@ -50,9 +52,14 @@ public BasicCollector(
   public KsqlVersionMetrics collectMetrics() {
     final KsqlVersionMetrics metricsRecord = new KsqlVersionMetrics();
     metricsRecord.setTimestamp(TimeUnit.MILLISECONDS.toSeconds(clock.millis()));
-    metricsRecord.setConfluentPlatformVersion(Version.getVersion());
+    metricsRecord.setConfluentPlatformVersion(cpVersion(AppInfoParser.getVersion()));
```

Comment:
Does `AppInfoParser.getVersion()` return the correct value? I see it gets the version from `kafka-version.properties` properties file which we don't have:
https://github.com/confluentinc/private-kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/utils/AppInfoParser.java